content/en/providers/check-in/sp/_index.md

```diff
@@ -521,15 +522,16 @@ using the `grant_type` value `refresh_token`:
 Example request:
 
 ```shell
-curl -X POST -u "${client_id}":"${client_secret}" \
--d "client_id={myClientID}" \
--d "client_secret={myClientSecret}" \
+curl -X POST "${TOKEN_ENDPOINT}" \
+-u "${CLIENT_ID}":"${CLIENT_SECRET}" \
 -d "grant_type=refresh_token" \
--d "refresh_token=${myRefreshToken}" \
--d "scope=openid%20email%20profile" \
-"https://aai.egi.eu/oidc/token" | python -m json.tool;
+-d "refresh_token=${REFRESH_TOKEN}" \
+-d "scope=openid%20email%20profile" | python -m json.tool;
```

Comment:
```suggestion
  -d "refresh_token=${REFRESH_TOKEN}" \
  -d "scope=openid%20email%20profile" | python -m json.tool;
```

Reply:
Resolved by commit [Add indent to curl data parameter](https://github.com/EGI-Federation/documentation/pull/332/commits/d4c45e1af19edd9a4d4cdaa64e897a218160bc87)